bundle/manifests/opentelemetry.io_opentelemetrycollectors.yaml

```diff
@@ -3703,9 +3703,9 @@ spec:
                   attached to Collector and Target Allocator pods.
                 type: object
               podSecurityContext:
-                description: PodSecurityContext holds pod-level security attributes
-                  and common container settings. Some fields are also present in container.securityContext.  Field
-                  values of container.
+                description: PodSecurityContext configures the pod security context
```

Comment:
@TylerHelmuth Running `make bundle` seemed to do the trick! I've checked in all the files except `config/manager/kustomization.yaml` as requested

Reply:
Sorry, I think the move was to run `make bundle` and then also run `make api-docs`. The tests were failing because I did the former but not the latter, and I think I've fixed that now.

@TylerHelmuth Can you please re-run the tests with https://github.com/open-telemetry/opentelemetry-operator/pull/1279/commits/8ee95e59cbef910b03da6bfbf42390c734beaf8c?